### ISSUE TITLE
Java sims DPI scaling

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/tasks#680]
> I have been using PhET applets and sims regularly in teaching undergraduate and graduate physical chemistry courses here at CU Boulder. I just purchased a new MS Surface tablet for teaching, and I found to my dismay that the JAVA based applets are displayed so small that they are unreadable and therefore unusable. The only recourse I have found so far is to change the display settings for the tablet (clunky!) to let them run. Do you have any other cure?

@phet-steele looked into options for disabling high-DPI scaling, but needs developer input.

**Comment by ariel-phet** [comment]
@oliver-phet I am marking this as low priority for the moment. Might be a better task for @arnabp 

This is a known issue (both @amyh-phet and I have seen this many times. 

It might be good to also let the person who wrote you know that we have a few more chem HTML5 sims in the works (SOM, atomic interactions (soon'ish), rutherford (just published), and MOTHA in the works)

**Comment by oliver-phet** [comment]
This likely isn't a task for @arnabp - I think it will take a developer. Looks like it requires modifying the Java manifest file and recompiling it.

**Comment by oliver-phet** [comment]
Got an email from Carl Wieman:

> I am trying to run the sims on my new surface pro 4 computer.  The new htlm5 sims look fine, but the old java sims are in little windows that only take up about 10% of the screen area.  Nothing I do will change the window size.  Changing the screen resolution does not help.
> Any ideas how to make them show bigger?  Very hard to use in my class this way.

May want to bump up the priority of this @ariel-phet ?

**Comment by samreid** [comment]
Stack overflow suggests these workarounds:
http://superuser.com/questions/988379/how-do-i-run-java-apps-upscaled-on-a-high-dpi-display

@ariel-phet suggested @phet-steele might try them and see if it works?  If so, we can give those instructions to our clients or post them on our website.

**Comment by samreid** [comment]
Also @kathy-phet suggests changing your screen resolution and see if that fixes things.

**Comment by samreid** [comment]
This blog reports that the problem will be solved in http://news.kynosarges.org/2015/06/29/javafx-dpi-scaling-fixed/ in Java release 8u60. Can you find out what version of Java Carl has and if an update to the latest version fixes it?

**Comment by samreid** [comment]
This reddit post claims it is fixed in Java 9:
https://www.reddit.com/r/java/comments/49ip70/java_swing_is_tiny_on_hidpi_displays_what_can_be/ which is available as an early release from https://jdk9.java.net/download/

This may be the best place for @phet-steele to start investigation.

**Comment by oliver-phet** [comment]
@samreid That sounds promising! Hopefully it resolves the issue for our sims!

**Comment by phet-steele** [comment]
@oliver-phet JRE 9 did miraculously fix the issue on my Win 10 laptop.

**Comment by phet-steele** [comment]
> This blog reports that the problem will be solved in Java release 8u60.

I'm not too confident about this, since I still exhibit the problem with 8u101

**Comment by ariel-phet** [comment]
Well lets assume it will solved in Java 9. Once Java 9 is release @phet-steele should check a few sims and then close this issue.

@oliver-phet perhaps we should have a help center entry for this issue?

**Comment by oliver-phet** [comment]
We might want to wait on publishing this to the Help Center until Java 9 is officially released. Oracle even says:

> These early access release downloads of the JRE and JDK are based on code available on OpenJDK at the time they were built and might not include the latest security patches.

If people email phethelp, I will certainly give them the scoop with the caveat that it may be a security concern.

**Comment by phet-steele** [comment]
http://www.java9countdown.xyz/

**Comment by veillette** [comment]
For your information, the previous website is not up to date.  See http://openjdk.java.net/projects/jdk9/ for the official release date.
At this point, it is due to be release on Sept 21. 2017.

**Comment by phet-steele** [comment]
@oliver-phet, Java 9 fixes this issue. Off to you for next steps.

**Comment by oliver-phet** [comment]
Help center has been updated: https://phet.colorado.edu/en/help-center/running-sims/windows

Currently the entry points to an Oracle page since the Java website still points to version 8.
